LambdaTH/TH-Build-Week-BE
# Ft map layout

#### What does this PR do?
```
Traverse graph and load rooms into the database
write traversed rooms details into room_details.py and paths rooms_map.py files respectively
```

#### Description of Task to be completed?
```
complete map layout
prevent duplicate entry to the database
```


#### How should this be manually tested?
in terminal write
```
python manage.py runserver 
```
go to `http://127.0.0.1:8000/data/` on browser
check that rooms are created in database at
`http://127.0.0.1:8000/admin`

#### Any background context you want to provide?


#### Screenshots (if appropriate)

#### Questions:
